### PR TITLE
Release v1.20.0

### PR DESCRIPTION
**Pull request by Anilm3**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,91 @@
...
+  loaded: ...
+```
+#### RASP operator versioning
+
+Finally, in order to distinguish multiple versions of our exploit prevention heuristics, RASP operators can now be versioned. Versioning is done with the following schema: `operator_name@version`, where the operator name is one of the existing RASP operators (`lfi_detector`, `ssrf_detector`, `sqli_detector`, `shi_detector`) and `version` consists of a single digit preceded by a `v`, e.g. `sqli_detector@v2`. 
+
+Operator versioning works as follows:
+- When the existing operator version is higher or equal to the required version, the available operator is compatible.
+- When the existing operator version is lower than the required version, the operator is incompatible.
+- When the operator is incompatible, the is silently skipped and added to the `skipped` section of the diagnostics.
```
Comment (estringana) [inline CHANGELOG.md]: Missing the word `operator` after the comma

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,91 @@
...
+
+In addition, this new release includes a new version of the `sqli_detector` operator, specifically `sqli_detector@v2`.
+
+### Release changelog
+#### Changes
+- Fingerprint regeneration based on availability of optional arguments ([#331](https://github.com/DataDog/libddwaf/pull/#331))
+- Expand detections per parameter ([#332](https://github.com/DataDog/libddwaf/pull/#332))
+- Extend exists operator to support key paths and negation ([#334](https://github.com/DataDog/libddwaf/pull/#334))
+- Negated scalar condition for matchers ([#335](https://github.com/DataDog/libddwaf/pull/#335))
+- Greater and lower than matchers
```
Comment (estringana) [inline CHANGELOG.md:78]: Missing link

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,91 @@
...
+
+### Release changelog
+#### Changes
+- Fingerprint regeneration based on availability of optional arguments ([#331](https://github.com/DataDog/libddwaf/pull/331))
+- Expand detections per parameter ([#332](https://github.com/DataDog/libddwaf/pull/332))
+- Extend exists operator to support key paths and negation ([#334](https://github.com/DataDog/libddwaf/pull/334))
+- Negated scalar condition for matchers ([#335](https://github.com/DataDog/libddwaf/pull/335))
+- Greater and lower than matchers  ([#336](https://github.com/DataDog/libddwaf/pull/336))
+- Support min_version and max_version on evaluation primitives and RASP operator versioning ([#343](https://github.com/DataDog/libddwaf/pull/343))
+- Introduce `sqli_detector@v2` ([#343](https://github.com/DataDog/libddwaf/pull/343))
```
Comment (estringana) [inline CHANGELOG.md:80]: Is this link ok?

Comment (Anilm3) [reply CHANGELOG.md:80]: Yes, I just wanted this change to be explicit

**Review by estringana — APPROVED** [review]

**Review by cataphract — APPROVED** [review]

**Pull request merged**